README_dataprep.md

```diff
@@ -5,18 +5,19 @@ To obtain transcripts for a particular video, do the following:
-  4. Run the command `python src/data_pipeline/prepare_data.py mentor\recordings\sessionN` from root directory of the git repository. *N* is the session number.
-  5. The program will first convert the video to audio and create **session1part1.wav.**, etc. based on what the video name is. The audio file has same name, except the extension
-  6. Then, the program will cut the audio into chunks, based on the timestamps provided. The chunks are stored inside **recordings/session1/audiochunks** and the chunks are named from **a0.wav** to **qm.wav** where *m* is the number of questions asked in the session.
-  7. Once the chunks have been obtained, the program will automatically call the IBM Watson method in `ibm_stt.py`.
-  8. The transcript will be automatically generated and stored as **transcript.csv** inside **recordings/session_n.**
-  9. Once the transcripts have been generated, put them on Google Sheets and manually edit them to make them coherent.
-  10. Work on generating the paraphrases for each question. Also assign Tags and Helper Tags to each question.
-  11. Combine the paraphrases and answers and add them to the **questions_paraphrases_answers** sheet inside MentorPAL/Dialogue on Google Drive. Do the same with the prompts and utterances and add them to **prompts_utterances**.
-  12. Download that sheet and **prompts_utterances** as XLSX and name them **questions_paraphrases_answers.xlsx** and **prompts_utterances.xlsx**. Place them inside MentorPAL/data on your local machine (the root, not inside your mentor folder)
-  13. Run `python src/data_pipeline/post_process_data.py example\path\to\mentor <start_session> <end_session>`. start_session and end_session are session numbers. Example, giving 1 4 will make the code run from session1 to session4. Giving 2 2 will make code run only for session2.
-  14. This will generate **classifier_data.csv**, **npceditor_data.xlsx** and **metadata.csv**. The first file is for use by the classifier. The second file should be imported into NPCEditor (More on this later) and the third file contains metadata about what was processed. If you have 5 sessions so far and then, you record a sixth session, you don't need to run the post_process_data.py from scratch. You can just run python src/post_process_data.py /example/path/to/mentor 6 6` and data from the sixth session will be appended to **classifier_data.csv** and **npceditor_data.xlsx**. **metadata.csv** will also be updated and you can add more new sessions like this. Yould still have to run prepare_data.py for the new sessions every time and have to do the manual work (paraphrases, tags, manual transcription cleanup) for each session.
-  15. Now, data for the ensemble classifier is ready and you can proceed to training the classifier and NPCEditor.
+  4. Activate **data-pipeline-env** virtualenv with `source data-pipeline-env/bin/activate`
```

Comment:
let's move this whole README to wherever the Makefile for this process is (subdir)